modules/nw-ingress-sharding-namespace-labels.adoc

```diff
@@ -15,6 +15,11 @@ a set of Ingress Controllers and when isolating traffic to a specific Ingress
 Controller. For example, company A goes to one Ingress Controller and company B
 to another.
 
+[WARNING]
+====
+If you deploy keeplived ingress VIP, do not deploy a non-default ingress controller with HostNetwork `endpointPublishingStrategy`. Doing so might cause issues. Use NodePort instead of HostNetwork `endpointPublishingStrategy`.
```

Comment:
We generally capitalize "Ingress" ([link](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress)) and a grep of "Keepalived" shows it also capitalized unless in a command. Plus a few points that hopefully help clarify the note a bit more:
```suggestion
If you deploy the Keepalived Ingress VIP, do not deploy a non-default Ingress Controller with value `HostNetwork` for the `endpointPublishingStrategy` parameter. Doing so might cause issues. Use value `NodePort` instead of `HostNetwork` for `endpointPublishingStrategy`.
```

Reply:
acked